src/test/scala-2/chiselTests/BoringUtilsSpec.scala

```diff
@@ -245,7 +139,7 @@ class BoringUtilsSpec extends AnyFlatSpec with Matchers with LogUtils with FileC
       BoringUtils.bore(bar.a_wire)
     }
     val e = intercept[Exception] {
-      circt.stage.ChiselStage.emitCHIRRTL(new Foo, args)
+      circt.stage.ChiselStage.emitCHIRRTL(new Foo, Array("--throw-on-first-error"))
```

Comment:
```suggestion
      circt.stage.ChiselStage.emitCHIRRTL(new Foo, args)
```
nit unnecesary change

Reply:
Fixed in https://github.com/chipsalliance/chisel/pull/4852/commits/e174a52d42ce766208bc06d0449de46060409441. This is technically single use as the other use site doesn't depend on it. I'll clean this up in a later commit.